Resfeber/View Controllers/MainTabBarController.swift

```diff
@@ -9,6 +9,28 @@
 import UIKit
 
 class MainTabBarController: UITabBarController {
+    
+    // MARK: - Properties
+    
+    fileprivate let profileButton: UIBarButtonItem = {
+        let buttonDiameter: CGFloat = 32
+        let button = UIButton(type: .system)
+        button.setDimensions(height: buttonDiameter, width: buttonDiameter)
+        button.layer.cornerRadius = buttonDiameter / 2
+        button.layer.masksToBounds = true
+        button.layer.borderWidth = 2
+        button.layer.borderColor = UIColor.Resfeber.red.cgColor
```

Comment:
I would suggest trying something like changing the button tintColor to red - see what that may look like